pkgs/development/libraries/qscintilla/default.nix

```diff
@@ -1,33 +1,30 @@
 { stdenv, lib, fetchurl, unzip
-, qt4 ? null, qmake4Hook ? null
-, withQt5 ? false, qtbase ? null, qtmacextras ? null, qmake ? null
+, withQt5 ? true, qtbase, qtmacextras ? null
+, qmake
 , fixDarwinDylibNames
 }:
 
 let
-  pname = "qscintilla-qt${if withQt5 then "5" else "4"}";
-  version = "2.11.6";
+  pname = "qscintilla-qt5";
+  version = "2.13.1";
```

Comment:
Put this few lines below instead of `inherit pname version;` and you can drop the whole `let` block.